pkg/scyllaclient/client_scylla_test.go

```diff
@@ -229,20 +229,22 @@ func TestClientDescribeRing(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if len(ring.Tokens) != 6*256 {
-		t.Fatal(len(ring.Tokens))
+	if len(ring.ReplicaTokens[0].Ranges) != 6*256 {
```

Comment:
Where is the value "6*256" taken from ?

Reply:
#nodes * default amount of vnodes per node in scylla.yaml.